.github/workflows/numpy.yml

```diff
@@ -22,10 +22,12 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        python -m pip install numpy==1.26.2
+        python -m pip install array-api-strict
         python -m pip install -r requirements.txt
     - name: Run the test suite
       env:
-        ARRAY_API_TESTS_MODULE: numpy.array_api
+        ARRAY_API_TESTS_MODULE: array_api_strict
       run: |
-        pytest -v -rxXfE --ci --skips-file numpy-skips.txt
+        pytest -v -rxXfE --skips-file numpy-skips.txt array_api_tests/
```

Comment:
This is the skips file I use in array-api-strict. https://github.com/data-apis/array-api-strict/blob/main/array-api-tests-xfails.txt. There's one special case test that fails in NumPy and one test that's a bug in the test suite.